src/pages/atomicdex/api/v20-dev/lightning/payments/index.mdx

```diff
@@ -0,0 +1,598 @@
+
+
+### Request Parameters
+
+| Parameter            | Type    | Description                                                                                                                           |
+|----------------------|---------|---------------------------------------------------------------------------------------------------------------------------------------|
+| coin                 | string  | Ticker of the coin to query.                                                                                                          |
+| payment_hash         | string  | The hexidecimal string which identifies a invoice. The `payment_hash` is returned from a `lightning::payments::send_payment` request. |
+
+
+#### 📌 Example
+
+<CodeGroup title="" tag="POST" label="lightning::payments::get_payment_details">
+```json {{ mm2MethodDecorate : true }}
+{
+    "method": "lightning::payments::get_payment_details",
+    "userpass": "MM2_RPC_PASSWORD",
+    "mmrpc": "2.0",
+    "params": {
+        "coin": "tBTC-lightning",
+        "payment_hash": "414f9b3524fc4e48c99f2723952732d8bc2eba1b35ce3bf2a70f5144b40f599e"
+    },
+    "id": 1
+}
+```
+</CodeGroup>
+
+
+<CollapsibleSection expandedText='Hide Response' collapsedText='Show Response'>
+```json
+{
+	"mmrpc": "2.0",
+	"result": {
+		"payment_details": {
+			"payment_hash": "c4e25cc10d77e3cd5f3d2af7b14ad72f123b2a5021bd6705c0b8ee8386bdeceb",
+			"payment_type": {
+				"type": "Outbound Payment",
+				"destination": "0348cc1a9479697cd52db445ea74149ad40bb01bb2045a3e8acba21b70f94ab7cf"
+			},
+			"description": "1 Blokaccino",
+			"amount_in_msat": 1000000,
+			"fee_paid_msat": 1803,
+			"status": "succeeded",
+			"created_at": 1684081413,
+			"last_updated": 1684081419
+		}
+	},
+	"id": 762
+}
+```
+</CollapsibleSection>
+
+
+<CollapsibleSection expandedText='Hide Error Responses' collapsedText='Show Error Responses'>
+### NoSuchPayment (payment hash not found)
+```
+{
+	"mmrpc": "2.0",
+	"error": "Payment with hash: 414f9b3524fc4e48c99f2723952732d8bc2eba1b35ce3bf2a70f5144b40f599e is not found",
+	"error_path": "get_payment_details",
+	"error_trace": "get_payment_details:75]",
+	"error_type": "NoSuchPayment",
+	"error_data": "414f9b3524fc4e48c99f2723952732d8bc2eba1b35ce3bf2a70f5144b40f599e",
+	"id": 762
+}
+```
+
+
+### InvalidRequest (payment_hash not a hash string)
+```
+{
+	"mmrpc": "2.0",
+	"error": "Error parsing request: invalid value: string '', expected a hash string",
+	"error_path": "dispatcher",
+	"error_trace": "dispatcher:109]",
+	"error_type": "InvalidRequest",
+	"error_data": "invalid value: string '', expected a hash string",
+	"id": 762
+}
+```
+</CollapsibleSection>
+
+
+## List Payments by Filter {{label : 'lightning::payments::list_payments_by_filter', tag : 'API-v2'}}
+
+The `lightning::payments::list_payments_by_filter` method returns a list of payments and invoices for a given coin which match the given filter.
```

Comment:
returns a list of payments only (sent or received), doesn't return invoices.

Reply:
thanks, fixed